src/common/utils.ts

```diff
@@ -146,7 +145,7 @@ function convertKeysFromSnakeCaseToCamelCase(obj: any): any {
 }
 
 function removeUndefined<T extends object>(obj: T): T {
-  return _.omitBy(obj, _.isUndefined) as T
+  return _.omitBy(obj, value => value === undefined) as T
```

Comment:
I would have created the function isUndefined in the Utils to harmonize the undefined checks. The reason for this suggestion is that today, based on the branch develop, there are 10 calls to `isUndefined` but 95 occurrences of `== undefined` (without the test folder) However, I will question the creation of this function because most of the cases can be solved by a truthy/falsy check.

```javaScript
function isUndefined(value){
  return undefined === value
}

//...  
return _.omitBy(obj, isUndefined) as T
//...
```

Reply:
A truthy/falsy check isn't always the right solution, because it'll return the same if it's `0` or `''` or `undefined`, which is often not desirable. It's safer to use `=== undefined`.

Reply:
i would in this case vote for:

```javascript
return _.omitBy(obj, value => value == null) as T
```

because that predicate will only be true for values of `null` or `undefined`. I like to save `=== undefined` for cases when I _REALLY_ want to check exclusively for `undefined`. what do you think?